tests/math/test_matrix_manipulation.py

```diff
@@ -818,3 +838,103 @@ def test_prod_matrices(self):
         assert final_wires == expected_wires
         assert qml.math.allclose(reduced_mat, expected_matrix)
         assert reduced_mat.shape == (2**5, 2**5)
+
+
+@pytest.mark.parametrize("ml_framework", ml_frameworks_list)
+class TestPartialTrace:
+    """Unit tests for the partial_trace function."""
+
+    @pytest.mark.parametrize("c_dtype", dtypes)
+    def test_single_density_matrix(self, ml_framework, c_dtype):
+        """Test partial trace on a single density matrix."""
+        # Define a 2-qubit density matrix
+        rho = qml.math.asarray(
+            np.array([[[1, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0]]]), like=ml_framework
+        )
+
+        # Expected result after tracing out the second qubit
+        expected = qml.math.asarray(np.array([[[1, 0], [0, 0]]]), like=ml_framework)
+
+        # Perform the partial trace
+        result = qml.math.quantum.partial_trace(rho, [0], c_dtype=c_dtype)
+        assert np.allclose(result, expected)
```

Comment:
you could use `qml.math.allclose` just to be fully ML interface agnostic :)

Reply:
added it in!